src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,20 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
you have some problems with fonts. For example:

.card__code should have font-weight 400
.card__price-amount should have color #060b35
etc.